registry/api/v2/errors.go

```diff
@@ -32,6 +32,17 @@ var (
 		HTTPStatusCode: http.StatusBadRequest,
 	})
 
+	// ErrorCodeRangeInvalid is returned when uploading a blob if the provided
+	// content range is invalid.
+	ErrorCodeRangeInvalid = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:   "RANGE_INVALID",
+		Message: "invalid content range",
+		Description: `When a layer is uploaded, the provided range will be
+		checked against the uploaded chunk. If the range is out of order,
+		this error will be returned.`,
```

Comment:
Nit (I see there's other places that should be updated as well); try to avoid using future tense;

```suggestion
		Description: `When a layer is uploaded, the provided range is
		checked against the uploaded chunk. If the range is out of order,
		this error is returned.`,
```

Perhaps a more active voice would be even better ("This error is returned if ...")

Reply:
updated, thanks.